src/docs/common_tasks/list_targets.md

```diff
@@ -6,33 +6,29 @@ You want to find out which Pants targets are currently available in your project
 
 ## Solution
 
-Use the `list` goal and specify a folder containing a `BUILD` file. This command, for example, would show all targets available in `myproject/BUILD`:
+Use the `list` goal and specify a folder containing a `BUILD` file. This command, for example, would show all targets available in `src/python/myproject/example/BUILD`:
 
 ```bash
-$ ./pants list myproject/:
+$ ./pants list src/python/myproject/example:
 ```
 
-This command would show all targets available in the `myproject` directory *as well as in all subdirectories*:
+This command would show all targets available in the `src/python/myproject` directory *as well as in all subdirectories*:
 
 ```bash
-$ ./pants list myproject/::
+$ ./pants list src/python/myproject::
 ```
 
 Note the syntactical difference between the single colon and the double colon.
 
 The output from a `./pants list` invocation may look something like this:
 
 ```bash
-$ ./pants list server/:
+$ ./pants list server:
 server/server:analytics
 server/server:bin
 server/server:tests
 ```
 
-## Warning
-
-We do not recommend using `./pants list ::` unless you're working with a very small Pants project. This command will list *all* targets currently available in the entire project, which can be a very slow operation in larger projects.
-
```

Comment:
This is true for Twitter, but not true for most orgs like this repo and Toolchain. The worst that happens is they wait up to a minute and/or use `ctrl-c`.

Also V2 list made this less of a concern afaict.